frontend/src/pages/journal/components/journal-card/journal-card.tsx

```diff
@@ -0,0 +1,25 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  title: string;
+  onClick: () => void;
+  isActive: boolean;
+};
+
+const JournalCard: React.FC<Properties> = ({ title, onClick, isActive }) => {
```

Comment:
We already have Card component, would it work to use it here?

Reply:
updated the card component